src/components/user/PersonalProfileEditForm.js

```diff
@@ -44,11 +45,21 @@ export default function PersonalProfileEditForm() {
       });
     } else {
       try {
+        // Create a FormData object to append the profile picture
+        const formDataForApi = new FormData();
+        formDataForApi.append('name', formData.name);
+        formDataForApi.append('email', formData.email);
+        formDataForApi.append('profilePicture', formData.profilePicture);
+
         // Call the backend API to update the user's name and email
-        const response = await axiosInstance.patch('/users/updateCurrentUser', {
-          name: formData.name,
-          email: formData.email,
-        });
+        const response = await axiosInstance.patch(
+          '/users/updateCurrentUser',
+          {
+            name: formData.name,
+            email: formData.email,
+          },
+          formDataForApi
```

Comment:
You can't add the `formDataForApi` here; unless you're also updating the `updateCurrentUser` endpoint to handle these values.

If you will be making the call to the code that's currently in your backend `imgur` branch, then this should make two separate API calls, one to the `updateCurrentUser` to update the name and email. And then one to `/users/:userId/uploadProfile` where you would send the profile image via form data.

Reply:
Now we have functionality in this way that we can't change only photo, we need to change name and email. What if the user wants to change only photo,we don't have this opportunity to give for user. What do you think , should add this functionality? 
<img width="366" alt="image" src="https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/88336329/aa43bb23-8962-4636-a255-5e2544aadc39">

Reply:
I tried to apply the second option, commit ef472280e946654d8b22b4e74fabf05a0f80d068
Could you please check?